app/controllers/v0/claim_documents_controller.rb

```diff
@@ -2,28 +2,36 @@
 
 require 'pension_burial/tag_sentry'
 require 'lgy/tag_sentry'
+require 'claim_documents/monitor'
+require 'lighthouse/benefits_intake/service'
+require 'pdf_utilities/datestamp_pdf'
 
 module V0
   class ClaimDocumentsController < ApplicationController
     service_tag 'claims-shared'
     skip_before_action(:authenticate)
+    before_action :load_user
 
     def create
-      Rails.logger.info "Creating PersistentAttachment FormID=#{form_id}"
+      uploads_monitor.track_document_upload_attempt(form_id, current_user)
 
-      attachment = klass.new(form_id:)
+      @attachment = klass.new(form_id:)
```

Comment:
make `attachment` a private attr_reader so the '@' is not needed everywhere

Reply:
we will need a attr_writer too, since we set some values for attachment. is that still preferred over continuing to use the instance variable directly?